src/ZoweUSSNode.ts

```diff
@@ -250,10 +308,37 @@ export class ZoweUSSNode extends vscode.TreeItem implements IZoweTreeNode {
      */
     public setEtag(etagValue): void {
         this.etag = etagValue;
+        /**
+         * helper method to change the node names in one go
+         * @param oldReference string
+         * @param revision string
+         */
+    }
+
     /**
-     * helper method to change the node names in one go
-     * @param oldReference string
-     * @param revision string
```

Comment:
Not sure why part of documentation jumped inside the body of the function 😄

Reply:
@Alexandru-Dumitru Fixed.